api/lib/database/helpers.js

```diff
@@ -30,9 +31,22 @@ const graphQLQuery = ({ hasura_url, hasura_admin_key }) => async (query) => {
   return data
 }
 
+function escapeValue(value) {
+  if (!value) return `""`
+  if (typeof value === 'object') {
+    const clone = JSON.parse(JSON.stringify(value))
+    Object.keys(clone).forEach((key) => {
+      clone[key] = escapeValue(clone[key])
+    })
+    return JSON.stringify(clone)
+  } else {
+    return `"${escape(value)}"`
+  }
+}
+
```

Comment:
We ended up doing this quite differently no? Strange that it doesn't show conflict. We should be careful with this, might overwrite